daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala

```diff
@@ -443,7 +443,7 @@ object Repl {
 
     parser.parseExprs[this.type](args.mkString(" ")) match {
 
-      case Left(error) =>
+      case Left(error @ _) =>
```

Comment:
What does this `x @ _` syntax do? I would have thought it would just be the same as `x`.

Unfortunately, it's very hard to search the web for this kind of syntax, so I have to ask.

Reply:
it is the same as `_`... but someone may say it is more readable because it includes a descriptive var name (unused).

Reply:
Sometimes it is helpful, and sometimes it is just noise; I have used one or the other based on my judgment in each case.